babel-watch.js

```diff
@@ -294,6 +299,18 @@ function restartApp() {
   childApp = app;
 }
 
+const delay = program.delay || 1000;
```

Comment:
Why is the default delay set to 1sec? I prefer it to be 0 (it's quite a frequent case that you only work on a single file and you don't save it more frequently than few time in a minute, I'd prefer not to enforce people to wait an additional second for their app to restart by default)